src/video-recorder/test-run-video-recorder.js

```diff
@@ -61,7 +61,9 @@ export default class TestRunVideoRecorder {
 
         return connectionCapabilities && connectionCapabilities.hasGetVideoFrameData;
     }
-
+    async isVideoEnabled () {
```

Comment:
Please try to separate class and object methods with an empty line here and in other places.